src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -342,7 +343,7 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 		this.monacoStyleContainer = append(header, $('.monaco-container'));
 		this.searchBox = this.instantiationService.createInstance(CodeEditorWidget, this.monacoStyleContainer,
 			mixinHTMLInputStyleOptions(getSimpleEditorOptions(), localize('searchExtensions', "Search Extensions in Marketplace")),
-			getSimpleCodeEditorWidgetOptions());
+			{ isSimpleWidget: true, contributions: [SuggestController] });
```

Comment:
I agree we don't need `SnippetController2` and `TabCompletionController`, but have you gone through what the other contributions (MenuPreventer, SelectionClipboard, ContextMenuController) of the simple editor have to offer to ensure we are not missing out on anything?

Reply:
- `MenuPreventer` stops `Alt+click` from toggling the menu bar visibility. We should have that.
- `SelectionClipboard` is a Linux specific thing that I can't seem to get to trigger on Windows. From what I can tell, it automatically adds whatever you select to your clipboard and pastes it on middle click. Not sure if this is something we want, given we select the search text for the user occasionally and this might overwrite their clipboard
- `ContextMenuController` allows you to right click in the search box. We should have that.